### PR TITLE
[new release] owl-zoo, owl-top, owl-base, owl and owl-plplot (0.7.2)

### PR DESCRIPTION
**Pull request by mseri**
OCaml Scientific and Engineering Computing - Zoo

- Project page: <a href="https://github.com/owlbarn/owl">https://github.com/owlbarn/owl</a>
- Documentation: <a href="https://owlbarn.github.io/">https://owlbarn.github.io/</a>

##### CHANGES:

* fourth order finite diff approx to grad
* changes to improve stability of sylv/discrete_lyap
* fix bug in concatenate function
* add mli for owl_base_linalg_generic
* Owl-base linalg routines: LU decomposition  (owlbarn/owl#465)
* bug fixes
* Update owl.opam

### CODE REVIEW

**Comment by mseri** [comment]
There will be expected linking failures with ubuntu and debian-unstable, and build failures on distributions that do not provide plplot

File: packages/owl/owl.0.7.2/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "alcotest" {with-test}
+  "base" {build & < "v0.13"}
```
Comment (kit-ty-kate) [inline packages/owl/owl.0.7.2/opam]: Is `owl` really incompatible with `base.v0.13` and `stdio.v0.13`?

Comment (mseri) [reply packages/owl/owl.0.7.2/opam]: Apparently not. Just tested locally. Thanks!

**Comment by mseri** [comment]
It worked. All the failures are the expected ones

**Comment by kit-ty-kate** [comment]
thanks!

**Pull request merged**